modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -538,6 +540,77 @@ export class Wallets implements IWallets {
       .result();
   }
 
+  /**
+   * Re-share wallet with existing spenders of the wallet
+   * @param walletId
+   * @param userPassword
+   */
+  async reshareOfcAccountWithSpenders(walletId: string, userPassword: string): Promise<void> {
```

Comment:
Are these methods meant to be exposed? If so they need to updated in the wallet interface in `iWallet.ts`